ext/com_dotnet/com_misc.c

```diff
@@ -50,7 +50,6 @@ PHP_COM_DOTNET_API void php_com_wrap_dispatch(zval *z, IDispatch *disp,
 	obj = emalloc(sizeof(*obj));
 	memset(obj, 0, sizeof(*obj));
 	obj->code_page = codepage;
-	obj->ce = php_com_variant_class_entry;
 	obj->zo.ce = php_com_variant_class_entry;
```

Comment:
It seems to me this is superfluous as well.

Reply:
Indeed, this is set by zend_object_std_init below